mixnode/src/commands/upgrade.rs

```diff
@@ -184,12 +186,12 @@ fn pre_090_upgrade(from: &str, config: Config) -> Config {
     upgraded_config
 }
 
-fn minor_010_upgrade(
+fn minor_0_10_upgrade(
     config: Config,
     _matches: &ArgMatches,
     config_version: &Version,
     package_version: &Version,
-) -> Config {
+) -> Result<Config, (Version, String)> {
```

Comment:
I like the `upgrade` command returning a `Result`, but I'm not a fan of this tuple error type. Perhaps some proper `UpgradeError` enum would be better?

Reply:
Do you mean a type alias for the error type? If so, then I agree, it would look better, I can add it in a following commit.